deps/wazuh_testing/wazuh_testing/qa_ctl/provisioning/wazuh_deployment/WazuhDeployment.py

```diff
@@ -27,7 +28,7 @@ class WazuhDeployment(ABC):
         hosts (string): Group of hosts to be deployed.
         server_ip (string): Manager IP to let agent get autoenrollment.
     """
-    def __init__(self, installation_files_path, configuration=None, inventory_file_path='/tmp/inventory.yaml',
+    def __init__(self, installation_files_path, inventory_file_path, configuration=None,
```

Comment:
Why you have deleted the default `inventory_file_path` value? I think that you just had to replace `/tmp/` to `gettempdir()`

Reply:
I have removed the default value, because at this point the inventory has already been created (with a random name), so it is mandatory to receive as a parameter a path with the inventory, because a default value would not be indicating anything since the name of the inventory is generated randomly.